package.json

```diff
@@ -25,12 +28,22 @@
     "build:docs": "cd docs && npm install && npm run build:html && cd .. && cp docs/*.html docs/*.css out/docs/",
     "build:spec": "ecmarkup spec.html out/index.html",
     "prebuild": "mkdirp out && mkdirp out/docs",
+    "pretty": "prettier --write docs/cookbook/*.*js polyfill/lib/*.*js polyfill/test/*.*js 'polyfill/test/**/*.*js'",
     "build": "npm run build:javascript && npm run build:docs && npm run build:spec"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/tc39/proposal-temporal.git"
   },
   "author": "Ujjwal Sharma <usharma@igalia.com>",
-  "license": "SEE LICENSE IN https://tc39.github.io/ecma262/#sec-copyright-and-software-license"
+  "license": "SEE LICENSE IN https://tc39.github.io/ecma262/#sec-copyright-and-software-license",
+  "prettier": {
```

Comment:
I’d suggest running prettier only via eslint, that way you also turn off conflicting eslint rules.